packages/main/src/Table.js

```diff
@@ -72,6 +112,20 @@ const metadata = {
 		showNoData: {
 			type: Boolean,
 		},
+
+		/**
+		 * Defines if additonal row will be displayed at the bottom of the table.
```

Comment:
I think this description is too technical (the implementation is with a row) but anyway we'll change this text once we add the growing feature as it would need to say "shows a more button or activates the even on scroll" then. I would suggest for now:

Defines if a "More" button will be displayed at the bottom of the table.
Pressing this button will fire the <code>load-more</code> event.